MODULE.bazel

```diff
@@ -0,0 +1,8 @@
+"""SPIRV-Headers"""
+
+module(
+    name = "spirv_headers",
+    version = "0.0.0",
```

Comment:
Please note that this value is just a placeholder.

Reply:
Where do you expect to use this? Should we have a more significant value for `version`?

Reply:
This will get used by the Bazel Central Registry (BCR) where this version will be overwritten with something to uniquely identify the version. The guidance for repositories that don't do semver releases is to use `0.0.0-YYYMMDD-{commit_hash}`. Bazel consumers will then specify the same version in their `MODULE.bazel` file. If there is an actual version number for these headers then that could be added here but I didn't want to increase the maintenance burden of these files and figured it would be fine to correctly reflect the version whenever someone wants a new release published to BCR.

Does that answer your question?

Reply:
Yup. Thanks. I was not too familiar with how the BCR works.